### PR TITLE
Update section titles and language for student readability

### PR DESCRIPTION
**Pull request by rrcobb**
Updated language around 'Three Pillars' to reflect that it's a Flatiron term, not an industry standard

Titles and headings sounded stilted (possibly based on the curriculum team workflow and patterns, not clarity for students?) 

Some minor content updates for clarity.

Changes:
- title and section names to more 'expected' names for students
- add bullet points to break down paragraphs
- rephrasing some paragraphs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
 - Identify the "Three Pillars of Web Programming"
+- Recognize web programming in an example
 
 ## Introduction
 
-"Front-End Web Programming" is a phrase that gets used in a lot of places by a
-lot of people. In this lesson we're going to create a common definition of "web
-programming" and show one example that demonstrates all of "web programming's"
-parts.
+The phrase "Front-End Web Programming" is used in different ways by different
```
Comment (sgharms) [inline README.md:11]: @rrcobb

* We hard wrap at 80col
* Delete trailing whitespace

You might want to add those plugins or rules to your editor of choice.

Per our [style guide](https://github.com/learn-co-curriculum/curriculum-team/blob/master/style_guide.md)

File: README.md [hunk 2]
```diff
@@ -1,33 +1,34 @@
...
+### Defining Web Programming
 
 Web programming, at its heart, is:
 
-> Creating documents with HTML, styling / positioning the document's content
-> with CSS, and using JavaScript to provide interactivity / notify remote
-> servers.
+> * Creating documents with HTML
+> * Styling the document's content with CSS
+> * Adding action and interactivity with Javascript
```
Comment (sgharms) [inline README.md:22]: It should be 

* creating / styling
* interactivity (eventing)
* server comms

File: README.md [hunk 3]
```diff
@@ -1,33 +1,34 @@
...
-lot of people. In this lesson we're going to create a common definition of "web
-programming" and show one example that demonstrates all of "web programming's"
-parts.
+The phrase "Front-End Web Programming" is used in different ways by different
+people. In this lesson we're going to choose a definition of "web
+programming" and show an example that demonstrates the different parts of "web programming",
+so that we know our objective for the next set of lessons.
 
-### Define Web Programming
+### Defining Web Programming
```
Comment (sgharms) [inline README.md:16]: https://github.com/learn-co-curriculum/curriculum-team/blob/master/style_guide.md#learning_goals

File: README.md [hunk 4]
```diff
@@ -68,16 +75,18 @@ web programmers.
...
 
 Web Programming is creating documents with HTML, styling / positioning the
-documents' content with CSS and updating that content and servers based on
-events using JavaScript. We can break down web programming into three pillars
-that involve working with the DOM, JavaScript eventing and communication with
-the server. Now that we've seen how these elements are connected in principle,
-we can now move on to seeing how they work together in practice.
+documents' content with CSS, and updating that content and servers based on
+events using JavaScript. We can break down the Javascript part of web 
+programming into three pillars that involve working with the DOM, Javascript
```
Comment (sgharms) [inline README.md:87]: https://github.com/learn-co-curriculum/curriculum-team/blob/master/style_guide.md#capitalization

**Review by sgharms — COMMENTED** [review]
@rrcobb lots of good changes here, very responsive, etc.

In a couple of places, I've flagged some style guide alignment ops.

I'm going to merge to a different branch and will notify you on the synthesized product.

File: README.md (hunk 1 above)
Comment (rrcobb) [reply README.md:11]: Cool, I was editing in the github web editor 😱 in lieu of a script (now in hand) to clone whole chunks of curriculum

File: README.md (hunk 2 above)
Comment (rrcobb) [reply README.md:22]: okay - there seemed to be two groups of three things: three 'frontend languages' html, css, js, and separately three pillars of fewp in js (dom manipulation, events, fetch). Maybe there is only one group of three things (the pillars)

File: README.md (hunk 3 above)
Comment (rrcobb) [reply README.md:16]: I suppose this speaks to the self-focused vs. student-focused language question. We definitely want to make sure that our content and our learning goals match up, but we're currently doing that at the sacrifice of the headings making sense to the reader

> # Define Web Programming

is a weird-sounding section title

**Comment by sgharms** [comment]
An initiative is coming out of the DS sector on a unified pattern for
writing LG's. We'll see where they resolve on that and standardize there.
Probably 1-2 weeks we'll see what the pattern is.

File: README.md (hunk 3 above)
Comment (sgharms) [inline README.md:16]: All top-level headings are `##` or `h2`.

**Pull request closed without merging**